docs/schema.md

```diff
@@ -387,6 +387,17 @@ The defined database prefixes and their "home" databases are:
         </ul>
       </td>
     </tr>
+    <tr>
+      <td><code>SUSE and openSUSE</code></td>
+      <td><a href="https://www.suse.com/support/security/">SUSE Security Landing page</a></td>
+      <td>
+        <ul>
+          <li>How to contribute: <a href="https://www.suse.com/support/security/contact/">https://www.suse.com/support/security/contact/</a></li>
+          <li>Source URL: <code>https://www.suse.com/support/update/</code></li>
+          <li>OSV Formatted URL: <code>https://ftp.suse.com/pub/projects/security/osv/</code></li>
+        </ul>
+      </td>
+    </tr>
```

Comment:
Would you be able to add some ecosystem definitions here: https://github.com/ossf/osv-schema/blob/516347895e3541e04db8166f9c83a22250f5b44d/docs/schema.md#affectedpackage-field also ?

There's a few other Linux distros in there that could potentially be useful to base off.

Reply:
I added stuff there